test/support/page_objects/header_page.js

```diff
@@ -42,8 +42,12 @@ export default class HeaderPage {
   }
 
   clickTimepicker() {
-    return PageObjects.common.findTestSubject('globalTimepickerButton')
-    .click();
+    // Wait for the page to render, otherwise we will get an error if Jenkins is too slow.
+    return PageObjects.common.sleep(200).then(() => {
```

Comment:
I think the timeout should be in clickNewSearch, after the click, so anyone calling clickNewSearch will have the appropriate delay and can call what they want after it.  At least I think thats where the error is coming from.  Does that make sense?

```
15:57:30.335: navigateToApp visualize
15:57:30.592: clickAreaChart
15:57:32.408: clickNewSearch
15:57:32.570: Set absolute time range from "2015-09-19 06:31:44.000" to "2015-09-23 18:31:44.000"
15:57:32.570: clickTimepicker
```